search-api/src/main/scala/no/ndla/searchapi/service/search/LearningPathIndexService.scala

```diff
@@ -44,14 +45,20 @@ class LearningPathIndexService(using
       domainModel: LearningPath,
       indexName: String,
       indexingBundle: IndexingBundle
-  ): Try[IndexRequest] = {
-    searchConverterService.asSearchableLearningPath(domainModel, indexingBundle).map { searchableLearningPath =>
-      val source = CirceUtil.toJsonString(searchableLearningPath)
-      indexInto(indexName)
-        .doc(source)
-        .id(domainModel.id.get.toString)
-        .versionType(EXTERNAL_GTE)
-        .version(domainModel.revision.map(_.toLong).get)
+  ): Try[Option[IndexRequest]] = {
+    if (domainModel.verificationStatus != CREATED_BY_NDLA) {
+      Success(None)
+    } else {
+      searchConverterService.asSearchableLearningPath(domainModel, indexingBundle).map { searchableLearningPath =>
+        val source = CirceUtil.toJsonString(searchableLearningPath)
+        Some(
+          indexInto(indexName)
+            .doc(source)
+            .id(domainModel.id.get.toString)
+            .versionType(EXTERNAL_GTE)
+            .version(domainModel.revision.map(_.toLong).get)
+        )
+      }
```

Comment:
Litt subjektivt, men kunne kanskje vært en `Option.when`?

```suggestion
    Option.when(domainModel.verificationStatus == CREATED_BY_NDLA) {
      searchConverterService.asSearchableLearningPath(domainModel, indexingBundle).map { searchableLearningPath =>
        val source = CirceUtil.toJsonString(searchableLearningPath)
          indexInto(indexName)
            .doc(source)
            .id(domainModel.id.get.toString)
            .versionType(EXTERNAL_GTE)
            .version(domainModel.revision.map(_.toLong).get)
      }
```